apps/settings/test/unit/panels/languages/language_test.js

```diff
@@ -4,15 +4,22 @@ suite('Languages > ', function() {
   var mockKeyboardHelper;
   var languages;
   var realL10n;
+  var realMozActivity;
+
+  // this will be used as the value of the deviceinfo.os setting
+  var ver = '' + Math.random();
```

Comment:
I prefer to make `ver` more readable like `OSVersion` or something else instead of an abbreviation. By doing so, we don't event need that comment !